common/src/test/java/tachyon/heartbeat/HeartbeatThreadTest.java

```diff
@@ -31,32 +32,61 @@
 
   private static final String THREAD_NAME = "heartbeat-thread-test-thread-name";
 
-  private static final int NUMBER_OF_THREADS = 1;
+  private static final int NUMBER_OF_THREADS = 10;
 
   private final ExecutorService mExecutorService = Executors.newFixedThreadPool(NUMBER_OF_THREADS);
 
-  @Before
-  public void setup() {
-    HeartbeatContext.setTimerClass(THREAD_NAME, HeartbeatContext.SCHEDULED_TIMER_CLASS);
-  }
-
   @Test
-  public void createHeartbeatThreadTest() throws Exception {
-    DummyHeartbeatExecutor executor = new DummyHeartbeatExecutor();
-    HeartbeatThread ht = new HeartbeatThread(THREAD_NAME, executor, 1);
+  public void concurrentHeartbeatThreadTest() throws Exception {
```

Comment:
Could you add the original test back? I think it would be valuable to know the single heartbeat test is working or not. Also, could you add a brief description of the test before the definition?